cc.spek.Spek.yml

```diff
@@ -0,0 +1,227 @@
+app-id: cc.spek.Spek
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: spek
+default-branch: stable
+rename-appdata-file: spek.metainfo.xml
+rename-desktop-file: spek.desktop
+rename-icon: spek
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/pkgconfig
+  - /share/man
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home:ro
```

Comment:
for an accoustic spectrum analyzer I don't see any sound access here.

Reply:
also why `filesystem=home` ?

Reply:
Spek only analyzes audio files from disk. Sound access is not needed for that.

Reply:
It seems like wxWidgets file chooser doesn't use portals (or maybe the code needs to be changed to use it but I don't know how). I checked MediaInfo, which also uses wxWidgets and it has `filesystem=host:ro` (
https://github.com/flathub/net.mediaarea.MediaInfo/blob/master/net.mediaarea.MediaInfo.json).